omnibus-nikos/automake.patch

```diff
@@ -0,0 +1,11 @@
+diff -bur automake-1.16/bin/automake.in  automake-1.16.new/bin/automake.in
```

Comment:
Can you add a comment above this line with the source of the patch and what it does?